File: api/Controllers/Models/Entities/Basket.cs
using System.ComponentModel.DataAnnotations;
using System;
using System.ComponentModel.DataAnnotations.Schema;
using System.Collections.Generic;

namespace IPKP___API.Controllers.Models.Entities
{
    public class Basket
    {
        [Key]
        [Column("Basket_ID")]
        public Guid Basket_ID { get; set; }

        [Column("Basket_Quantity")]
        public int Basket_Quantity { get; set; }

        //stock item foreign key
        public Guid? Stock_Item_ID { get; set; }
        public virtual Stock_Item Stock_Item { get; set; }

        //Customer foreign key
        public Guid? Customer_ID { get; set; }
        public virtual Customer Customer { get; set; }

        //public virtual ICollection<Customer> Customer { get; set; }
        
    }
}

File: admin/src/app/Models/basket.ts
import { Stock_Item } from 'src/app/Models/stockitem';
import { Customer } from './customer';

export class BasketItems {
    basket_ID! : Number
    basket_Quantity!: number;
    stock_Item_ID!: number
    stock_Image_ID!: number

    stock_Item!: Stock_Item;
    customer!: Customer
    price!: number;
}